DataDefinitions/ResourceBasedLocalizedEDName.cs

```diff
@@ -40,6 +40,13 @@ public string localizedName
                 string localizedName = resourceManager.GetString(basename) ?? basename;
                 return localizedName;
             }
+            set
+            {
+                if (localizedName == null)
```

Comment:
Edit: I think my reservation is that my design for the class envisages `localizedName` as a purely derived property. In fact I would now write it as:

```csharp
        public string localizedName => resourceManager.GetString(basename) ?? basename;
```

Hence the setter is broken from the start, as `localizedName` can only be null if `basename` is null, and that doesn't happen.

I really don't think `localizedName` should have a setter. I flies in the face of the class design.

It may be valid to indroduce a settable property `fallbackLocalizedName` and rewrite `localizedName` to:

```csharp
        public string localizedName => resourceManager.GetString(basename) ?? fallbackLocalizedName ?? basename;
```
But frankly I wonder if we are starting to over-engineer things here.

Reply:
If we just want to display the edname in the UI for materials without a definition then we don't need this.  fallbackLocalizedName is acceptable too.